addons/mail/static/src/messaging/component/follower/follower_tests.js

```diff
@@ -0,0 +1,436 @@
+odoo.define('mail.messaging.component.FollowerTests', function (require) {
+'use strict';
+
+const components = {
+    Follower: require('mail.messaging.component.Follower'),
+};
+const {
+    afterEach: utilsAfterEach,
+    afterNextRender,
+    beforeEach: utilsBeforeEach,
+    nextAnimationFrame,
+    pause,
+    start: utilsStart,
```

Comment:
I leave that discussion to @alexkuhn :p

Reply:
It was meant for him, I will wait his answer then.

Reply:
Some arguments against `utils.[...]`:
- why utils does it mean? e.g. when using `web.utils`, `web.test_utils`, `mail.messaging.utils`, and `mail.messaging.testUtils`, this becomes bothersome to create and name a variable for each case.
- `utils` modules contain pure functions (minus few exceptions like `patch()` that have some state, but still...), so it makes sense to use them without a "receiver" in the OOP world.
- using `.[...]` is prone to typos because don't have help of IDE/linter for languages with duck typing like JS.

In this case, we use rename because we want use a custom `start()` in this test suite, so that it does almost everything before scenario/assertions. I think the correct solution would be to rename the `start()` of messaging utils to something like `makeEnv()` and only does setup for OWL env, and then each test explicitly make components they want to test (could be a helper function in `beforeEach()`).

In short, I wouldn't change it in this PR yet, but at the moment we clean the helper `start()`.